### DISCUSSION TITLE
1.21.1 update?

### DISCUSSION THREAD

**Discussion by robschof** [MinecraftFreecam/Freecam#248] [Q&A]
Hello!  

I was wondering if there were plans to update freecam to 1.21.1?  

(Im not sure I am asking in the right spot.  Sorry if I'm not!)

Thanks!!

**Comment by MattSturgeon** [comment]
AFAIK the Freecam build for 1.21 should be compatible with 1.21.1

If it isn't please let me know! :grin:

**Reply by robschof** [reply]
oh interesting.  based on that I thought I would do another test.  CurseForge wont let me install Freecam 1.3 in my 1.21.1 modpack.  However, when I downloaded it from moderinth and put it in my mod folder manually it worked just fine.  Looks like I am all good!

(PS:  Thanks so much for being so responsive!!)
